build/gulpfile.vscode.web.js

```diff
@@ -202,7 +202,7 @@ function packageTask(sourceFolderName, destinationFolderName) {
 
 const compileWebExtensionsBuildTask = task.define('compile-web-extensions-build', task.series(
 	task.define('clean-web-extensions-build', util.rimraf('.build/web/extensions')),
-	task.define('bundle-web-extensions-build', () => extensions.packageLocalExtensionsStream(true, false).pipe(gulp.dest('.build/web'))),
+	task.define('bundle-web-extensions-build', () => extensions.packageAllLocalExtensionsStream(true, false).pipe(gulp.dest('.build/web'))),
```

Comment:
Not going to block the PR on this, but is building the native extensions have an effect in the web pipeline. Maybe we should skip the native ones ?

Reply:
In Microsoft Auth's case, it has web code as well... so it actually does need to be built for web.